internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -480,6 +481,23 @@ func (r *BMACReconciler) reconcileSpokeBMH(ctx context.Context, bmh *bmh_v1alpha
 		return reconcileError{err}
 	}
 
+	// Add detached annotation to hub BMH so that Ironic stops managing the host from
+	// the hub cluster. After spoke BMH is created, the host will be managed by the spoke
+	// cluster.
+	bmhAnnotations := bmh.ObjectMeta.GetAnnotations()
+	if bmhAnnotations[BMH_DETACHED_ANNOTATION] != "true" {
```

Comment:
I think my comments from https://github.com/openshift/assisted-service/pull/1540/files#r620048077 may apply to this PR too.

Reply:
It does. Will change it to check the annotation does not exist.